mri/tests/test_fourier_adjoint.py

```diff
@@ -113,18 +113,23 @@ def test_FFT(self):
         print(" FFT adjoint test passes")
 
     def test_NFFT_2D(self):
-        """Test the adjoint operator for the 2D non-Cartesian Fourier transform
+        """Test the adjoint operator for the 2D non-Cartesian Fourier
+        transform, with density compensator set to 1, to vet the code
+        path, the test is unchanged otherwise
```

Comment:
I think I would be happier with 2 tests one without density compensation set and the other with it set to one

Reply:
Well, I really dont want to add on to tests as currently the time is nearly 40 minutes.
So the 3D works without density compensator while for 2D it works with density compensation. 
This way the code paths are vetted in both the cases. 
In the end the test is actually on the adjoint property more than density compensation itself. I havent really gotten a test for density compensation itself and I want to get one for it in future and that would be more meaningful perhaps to add?

Reply:
I see, well then one test here makes sense.